ddtrace/contrib/langchain/patch.py

```diff
@@ -669,6 +669,72 @@ async def traced_chain_acall(langchain, pin, func, instance, args, kwargs):
     return final_outputs
 
 
+@with_traced_module
+def traced_lcecl_runnable_sequence(langchain, pin, func, instance, args, kwargs):
+    integration = langchain._datadog_integration
+    span = integration.trace(pin, "%s.%s" % (instance.__module__, instance.__class__.__name__), interface_type="chain")
+    log_inputs = {}
+    log_outputs = {}
+    try:
+        inputs = get_argument_value(args, kwargs, 0, "input")
+        if integration.is_pc_sampled_span(span):
```

Comment:
I know this isn't the change you introduced, but wondering, what is `is_pc_sampled_span` supposed to be doing? Why does it do its own sampler that's just a rate sampler?

Reply:
Is it supposed to actually be running sampling on a span and setting sampling priority tags?

Reply:
Yeah good question, I don't have too much context for this. cc @Yun-Kim, if you have any input here

Reply:
@ZStriker19  `is_pc_sampled_span()` doesn't have anything to do with submitting the span / sampling priority, it's a random rate sampler that determines whether or not we tag the prompt/completion. It's a bit of a weird name, open to any suggestions :)

Reply:
Interesting... it just does it at random then if there isn't a sampling priority already set then huh?